virttest/shared/scripts/virtio_console_guest.py

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python
 # -*- coding: utf-8 -*-
+#!/usr/bin/python
```

Comment:
Well, IMO it has to be the first line for taking effect.

Reply:
> Well, IMO it has to be the first line for taking effect.

Updated and thanks.